### PR TITLE
Add statsd metrics for number of taxons per level

### PR DESCRIPTION
**Pull request by koetsier**
Trello: https://trello.com/c/fNMNMaiz/277-visualise-number-of-taxons-at-each-level-in-grafana

### CODE REVIEW

File: app/services/metrics/taxonomy_metrics.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+require_relative '../metrics'
+
+module Metrics
+  class TaxonomyMetrics
```
Comment (cbaines) [inline app/services/metrics/taxonomy_metrics.rb]: All the metrics in this namespace are currently about the taxonomy, so maybe this class could be called something more specific, TaxonsPerLevelMetrics perhaps?

File: spec/services/metrics/taxonomy_metrics_spec.rb [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+require 'gds_api/test_helpers/content_store'
+
+include ::GdsApi::TestHelpers::ContentStore
+
+module Metrics
+  RSpec.describe TaxonomyMetrics do
+    describe '#count_taxons_per_level' do
+      before do
+        content_store_has_item('/', root_taxon.to_json, draft: true)
+        content_store_has_item('/taxons/root_taxon', multi_level_child_taxons.to_json, draft: true)
```
Comment (cbaines) [inline spec/services/metrics/taxonomy_metrics_spec.rb]: Might be good to use the newer "level one" terminology, especially given you went to the effort of making that change elsewhere.

File: app/services/metrics/taxonomy_metrics.rb (hunk 1 above)
Comment (koetsier) [reply app/services/metrics/taxonomy_metrics.rb]: fixed

File: spec/services/metrics/taxonomy_metrics_spec.rb (hunk 2 above)
Comment (koetsier) [reply spec/services/metrics/taxonomy_metrics_spec.rb]: fixed

**Pull request merged**